### PR TITLE
Create societe-francaise-degyptologie.csl

### PR DESCRIPTION
**Pull request by NicolasSouchon**
Create the societe-francaise-degyptologie style for the publications “Revue d’Égyptologie” and “Bulletin de la Société française d’égyptologie” according to the recommendations to authors: https://www.sfe-egyptologie.fr/index.php/ publications/recommendations-to-authors

### CODE REVIEW

File: societe-francaise-degyptologie.csl [hunk 1]
```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Société française d'égyptologie</title>
+    <title-short>SFE</title-short>
+    <id>http://www.zotero.org/styles/societe-francaise-degyptologie</id>
+    <link href="http://www.zotero.org/styles/societe-francaise-degyptologie" rel="self"/>
+    <link href="https://www.sfe-egyptologie.fr/index.php/publications/recommandations-aux-auteurs" rel="documentation"/>
```
Comment (bwiernik) [inline societe-francaise-degyptologie.csl:9]: If you started from an existing style, can you please add a "template" link to it here?

**Review by bwiernik — CHANGES_REQUESTED** [review]
Sorry on the delay! Did you start from an existing style? If so, can you add a template link, then we can merge this.

File: societe-francaise-degyptologie.csl (hunk 1 above)
Comment (NicolasSouchon) [reply societe-francaise-degyptologie.csl:9]: Hello, I created the style from scratch. Thanks for your help

File: societe-francaise-degyptologie.csl [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                    <else-if type="chapter" match="any">
+                      <text macro="container-title"/>
+                    </else-if>
+                  </choose>
+                  <group delimiter=" " prefix="(" suffix=")">
+                    <text macro="collection"/>
+                    <text macro="collection-number"/>
+                  </group>
+                </group>
+                <text macro="date"/>
```
Comment (bwiernik) [inline societe-francaise-degyptologie.csl:203]: There is no space or other delimiter coming between the container information and the date. I'm guessing that's not right. 

You add a group with the appropriate delimiter around this container+date section and the other places above

File: societe-francaise-degyptologie.csl [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+              </else>
+            </choose>
+          </else>
+        </choose>
+        <text macro="locator"/>
```
Comment (bwiernik) [inline societe-francaise-degyptologie.csl:208]: Similarly, it looks like there is no delimiter between the rest of the citation and the locator. Is that correct?

**Review by bwiernik — CHANGES_REQUESTED** [review]
One other question about missing delimiters. I noticed these by checking the test item output above from the test bot

File: societe-francaise-degyptologie.csl (hunk 2 above)
Comment (NicolasSouchon) [reply societe-francaise-degyptologie.csl:205]: There was missing a group around lines 162-164, a space should appear now

File: societe-francaise-degyptologie.csl (hunk 3 above)
Comment (NicolasSouchon) [reply societe-francaise-degyptologie.csl:210]: The comma used as a delimiter by the group started on line 117 is not enough?

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**